src/routes/public.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import Home from '../screens/Home';
+import Login from '../screens/Login';
+
+import { loginScreen } from '../constants/navigation/constants';
+import { homeScreen } from '../constants/navigation/constants';
```

Comment:
...e aqui bastaria importar o objeto (poderia também tornar o caminho mais simples)
```suggestion
import { routes } from '../constants/navigation';
```